src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -3594,6 +3594,14 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
 
   lazy val NoSymbol: NoSymbol = makeNoSymbol
 
+  /** A symbol representing a language feature, along with some useful information. */
+  case class LanguageFeature(sym: Symbol, name: String) {
+    def tpe = sym.tpe
+    val AnnotationInfo(_, List(
+          Literal(Constant(desc: String)),
+          Literal(Constant(required: Boolean))
+        ), _) = sym.getAnnotation(LanguageFeatureAnnot).get
+  }
```

Comment:
During the `scala` corpus [benchmark](https://scala-ci.typesafe.com/view/scala-bench/job/compiler-benchmark/1813/console), which has the language features defined in sources:

```
[error] 	Suppressed: java.util.NoSuchElementException: None.get
[error] 		at scala.None$.get(Option.scala:349)
[error] 		at scala.None$.get(Option.scala:347)
[error] 		at scala.reflect.internal.Symbols$LanguageFeature.<init>(Symbols.scala:3603)
[error] 		at scala.reflect.internal.Definitions$DefinitionsClass.getLanguageFeature(Definitions.scala:1222)
[error] 		at scala.reflect.internal.Definitions$DefinitionsClass$RunDefinitions.ImplicitConversionsFeature$lzycompute(Definitions.scala:1539)
[error] 		at scala.reflect.internal.Definitions$DefinitionsClass$RunDefinitions.ImplicitConversionsFeature(Definitions.scala:1539)
[error] 		at scala.tools.nsc.typechecker.Typers$Typer.typedDefDef(Typers.scala:2334)
[error] 		at scala.tools.nsc.typechecker.Typers$Typer.typedMemberDef$1(Typers.scala:5545)
[error] 		at scala.tools.nsc.typechecker.Typers$Typer.typed1(Typers.scala:5596)
```

Reply:
... yikes. It bootstrapped just fine (obviously). I'll take a look as to why, or just get rid of the commit if it's not worth it.

Reply:
The point of difference is that over in compiler-benchmark we jointly compile reflect/library/compiler sources, which seems to extend some weird corner cases with bootstrapping